impeller/entity/entity_unittests.cc

```diff
@@ -1393,5 +1393,99 @@ TEST_P(EntityTest, RRectShadowTest) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
+TEST_P(EntityTest, ColorMatrixFilterCoverageIsCorrect) {
+  // Set up a simple color background.
+  auto fill = std::make_shared<SolidColorContents>();
+  fill->SetPath(
+      PathBuilder{}.AddRect(Rect::MakeXYWH(0, 0, 300, 400)).TakePath());
+  fill->SetColor(Color::Coral());
+
+  // Set the color matrix filter.
+  FilterContents::ColorMatrix matrix = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+                                        1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
```

Comment:
Nit: For these and all of the other matrices, maybe lay these out in a 5x4 grid for easy reading? Often we use comments to coerce a formatter in situations like this. For example
```
FilterContents::ColorMatrix matrix = {
    1, 1, 1, 1, 1, //
    1, 1, 1, 1, 1, //
    1, 1, 1, 1, 1, //
    1, 1, 1, 1, 1, //
};
```

Reply:
Nice! I edited the matrices now.